fireblocks_sdk/sdk.py

```diff
@@ -57,18 +57,19 @@
 def handle_response(response, page_mode=False):
     try:
         response_data = response.json()
-    except:
+    except (AttributeError, ValueError):
         response_data = None
```

Comment:
what about other types of exceptions?

Reply:
Which other types of exceptions do you have in mind that would be appropriate to silence and automatically handle this way?

As it is, the wild except statement would catch all exceptions, including system-related errors that have nothing to do with the Fireblocks library itself (e.g. program could be terminated, out of memory, etc..). In such cases it is more appropriate to propagate the exception rather than silencing it this way.

Wild except statements are very discouraged in general and are considered bad practice in Python.